### PR TITLE
Import manifests for persistent-disk tutorial

### PR DESCRIPTION
**Pull request by ahmetb**
This tutorial's contents used to be served in a .zip on cloud.google.com,
now moving them here and updating. Major changes:

- use `Service` to do discovery instead of docker-links style env vars
- use Deployment instead of `Pod` deployment
- extract mysql root password to a `Secret` instead of embedding in YAML

Contact me for internal link to review the new tutorial if necessary.

Signed-off-by: Ahmet Alp Balkan <ahmetb@google.com>

### CODE REVIEW

**Comment by ahmetb** [comment]
@waprin can you please review this one?

File: wordpress-persistent-disks/mysql.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    spec:
+      containers:
+        - image: mysql:5.6
+          name: mysql
+          env:
+            - name: MYSQL_ROOT_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: mysql
+                  key: password
```
Comment (waprin) [inline wordpress-persistent-disks/mysql.yaml]: where does this secret come from?

Comment (ahmetb) [inline wordpress-persistent-disks/mysql.yaml]: @waprin it's created in the tutorial with `kubectl create secret`.

File: wordpress-persistent-disks/mysql-service.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: mysql
+  labels:
+    app: mysql
+spec:
+  type: ClusterIP
```
Comment (waprin) [inline wordpress-persistent-disks/mysql-service.yaml]: We usually just let it default to this? Makes sense and is cleaner to me but maybe being explicit is better, idk.

Comment (ahmetb) [inline wordpress-persistent-disks/mysql-service.yaml]: yeah I wanted to make it explicit, as the other wordpress-service.yaml has type:LoadBalancer.

File: wordpress-persistent-disks/mysql.yaml (hunk 1 above)
Comment (waprin) [inline wordpress-persistent-disks/mysql.yaml]: I don't see that in the tutorial linked...

**Comment by waprin** [comment]
LGTM

File: wordpress-persistent-disks/mysql.yaml (hunk 1 above)
Comment (mike-trewartha) [inline wordpress-persistent-disks/mysql.yaml]: agreed with @waprin - the process to create a secret is not in the tutorial.

Comment (ahmetb) [inline wordpress-persistent-disks/mysql.yaml]: @mike-trewartha for now please follow the https://cloud.google.com/container-engine/docs/tutorials/persistent-disk/. The new version of the tutorial (which this PR adds) is not published yet.

**Pull request merged**